Makefile

```diff
@@ -411,3 +411,14 @@ container-build-and-push: container-build container-push ## Build and push all t
 .PHONY: cluster-functest 
 cluster-functest: ginkgo ## Run e2e tests in a real cluster
 	./hack/functest.sh $(GINKGO_VERSION)
+
+.PHONY: super-linter
+super-linter: ## Runs super linter locally (Supported Linters -> https://github.com/github/super-linter#supported-linters)
+	docker run --rm \
+		-e RUN_LOCAL=true \
+		-e USE_FIND_ALGORITHM=true \
+		-e IGNORE_GITIGNORED_FILES=true \
+		-e FILTER_REGEX_EXCLUDE="/vendor/" \
+		-v $$(pwd):/tmp/lint \
+		-w /tmp/lint \
+		github/super-linter:v4
```

Comment:
Correct, but I am not sure we want that since it only includes rust, dotenv, armttk, pwsh, and c# linters.
We mainly want/need `golangci-lint` for Golang which is missing in the slim version.

Reply:
IIUC the slim version has all the linters except the following

    rust linters
    dotenv linters
    armttk linters
    pwsh linters
    c# linters

Reply:
oh geez, I have blocked myself of viewing the following sentence ` but removes the following:`. Thanks for being persistent 👍🏻